.github/workflows/update-beamos-tags.yml

```diff
@@ -0,0 +1,61 @@
+name: Update tags on mrbeam/mrb3-beamOS
+
+on:
+  workflow_call:
+
+env:
+  PERSONAL_ACCESS_TOKEN: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
+
+
+jobs:
+  update-tags:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout mrb3-beamOS
+        uses: actions/checkout@v2
+        with:
+            repository: mrbeam/mrb3-beamOS
+            path: mrb3-beamOS
+            token: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
+            fetch-depth: 0 # Fetch all history for all tags and branches
+
+      - name: Update mrbeam-git-tags.inc
+        run: |
+          cd mrb3-beamOS
+          git config --global user.email "softwareautomations@mr-beam.org"
```

Comment:
I created two variables for these, so that they're available throughout the organization
- ${{ env.SW_AUTOMATIONS_EMAIL  }}
- ${{ env.SW_AUTOMATIONS_NAME  }}
 (This is how I created them: https://docs.github.com/en/actions/learn-github-actions/variables)

Reply:
for later